test/gurka/test_time_tracking.cc

```diff
@@ -0,0 +1,116 @@
+#include "baldr/graphreader.h"
+#include "baldr/location.h"
+#include "gurka.h"
+#include "loki/search.h"
+#include "proto/api.pb.h"
+#include "sif/costfactory.h"
+#include "thor/pathalgorithm.h"
+
+#include <gtest/gtest.h>
+
+using namespace valhalla;
+namespace dt = valhalla::baldr::DateTime;
+
+TEST(TimeTracking, make) {
+
+  // build a very simple graph
+  const std::string ascii_map = R"(A----B)";
+  const gurka::ways ways = {{"AB", {{"highway", "trunk"}}}};
+  const auto layout = gurka::detail::map_to_coordinates(ascii_map, 100);
+  auto map = gurka::buildtiles(layout, ways, {}, {}, "test/data/gurka_time_tracking_make",
+                               {{"mjlonir.timezone", "/path/to/timezone.sqlite"}});
+
+  // need to access the tiles
+  baldr::GraphReader reader(map.config.get_child("mjolnir"));
+
+  // get some loki results
+  sif::CostFactory<sif::DynamicCost> factory;
+  factory.RegisterStandardCostingModels();
+  Options options;
+  options.set_costing(Costing::none_);
+  auto costing = factory.Create(options);
+  auto found = loki::Search({baldr::Location(map.nodes.begin()->second)}, reader, costing);
+  auto* location = options.add_locations();
+  baldr::PathLocation::toPBF(found.begin()->second, location, reader);
+
+  // no time
+  auto ti = thor::TimeInfo::make(*location, reader);
+  ASSERT_EQ(ti, thor::TimeInfo{});
+  ASSERT_FALSE(location->has_date_time());
+
+  // current time (technically we could fail if the minute changes between the next 3 lines)
+  location->set_date_time("current");
+  ti = thor::TimeInfo::make(*location, reader);
+  auto now_str = dt::iso_date_time(dt::get_tz_db().from_index(1));
+  auto lt = dt::seconds_since_epoch(now_str, dt::get_tz_db().from_index(1));
```

Comment:
What does `lt` mean? "lunixtime? :)

Reply:
local_time, which is the value it gets placed into in the brace intializer